webviz_ert/controllers/multi_response_controller.py

```diff
@@ -196,4 +202,34 @@ def _generate_plot(ensemble_id: str, color: str) -> Optional[ResponsePlotModel]:
             for observation in plot._observations:
                 fig.add_trace(observation.repr)
         fig.update_layout(assets.ERTSTYLE["figure"]["layout"])
+        fig.update_layout({"xaxis": {"title": {"text": x_axis_label}}})
+        fig.update_layout(assets.ERTSTYLE["figure"]["layout-value-y-axis-label"])
         return fig
+
+
+def get_index_type_with_sample(
```

Comment:
The function returns multiple return variables, we should try to avoid that. Also not really sure what this function adds, it seems that what it returns is only used by the new function `axis_label_for_type` and also the logic is not really that complex so I guess it can just be part of `axis_label_for_type`.